mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -756,6 +883,7 @@
         <AutoCastRange value="6"/>
         <AutoCastValidatorArray value="Casterhas10EnergyorCasterisBatteryOvercharged"/>
         <AutoCastValidatorArray value="UnitOrAttackingStructure"/>
+        <AutoCastValidatorArray value="NotHaveBatteryCooldownBehavior"/>
```

Comment:
Fixed an issue where repeatedly issuing a Stop command could increase the shield regen rate of a Shield Battery's target

Adds the requirement for not having the cooldown behavior to the auto-cast